packages/rest-client-jsonapi/src/services/services.test.ts

```diff
@@ -104,73 +115,200 @@ describe("Testing CRUD operations against Hatchify backend", async () => {
+    server.close()
+  })
+
+  // Note: This test does not spin up a Postgres server on its own. You must have one running locally in order for this test to pass.
+  it("successfully runs CRUD operations on schemas with different namespaces", async () => {
+    const app = new Koa()
+    const hatchedKoa = hatchifyKoa(
+      { Feature_Article, Admin_User },
+      {
+        prefix: `/${testBackendEndpointConfig.api}`,
+        database: {
+          dialect: "postgres",
+          host: "localhost",
+          port: Number(process.env.PG_DB_PORT),
+          username: "postgres",
+          password: "postgres",
+          database: "postgres",
+          logging: false,
+        },
+      },
+    )
+    app.use(cors())
+    app.use(hatchedKoa.middleware.allModels.all)
+    await hatchedKoa.createDatabase()
+    const server = app.listen(3012) // TODO determine why unique port numbers are needed across tests in this file (otherwise, tests are liable to fail): https://bitovi.atlassian.net/browse/HATCH-415
+    const jsonApi = jsonapi(
+      `http://localhost:3012/${testBackendEndpointConfig.api}`,
+      {
+        Feature_Article: {
+          ...Feature_Article,
+          endpoint: `${testBackendEndpointConfig.namespacedSchemaSegment}`,
+        },
+        Admin_User: {
+          ...Admin_User,
+          endpoint: `${testBackendEndpointConfig.adminUserNamespaceSegment}`,
+        },
+      },
+    )
+    const hatchedReactRest = hatchifyReactRest(jsonApi)
+
+    await hatchedReactRest.Admin_User.createOne({
+      attributes: {
+        name: "Juno",
+      },
+    })
+
+    await hatchedReactRest.Feature_Article.createOne({
+      attributes: {
+        author: "John Doe",
+        tag: "Hatchify",
+      },
+    })
+
+    const [adminUsers] = await hatchedReactRest.Admin_User.findAll({})
+    const [adminUser] = adminUsers
+    const { id: userId } = adminUser
+    expect(adminUsers.length === 1)
+    expect(adminUser).toEqual({
+      id: userId,
+      __schema: "Admin_User",
+      name: "Juno",
+    })
+
+    const [featureArticles] = await hatchedReactRest.Feature_Article.findAll({})
+    const [featureArticle] = featureArticles
+    const { id } = featureArticle
+    expect(featureArticles.length === 1)
+    expect(featureArticle).toEqual({
+      id,
+      __schema: "Feature_Article",
+      admin_UserId: null,
+      author: "John Doe",
+      tag: "Hatchify",
+    })
+
+    await hatchedReactRest.Admin_User.updateOne({
+      id: userId,
+      attributes: {
+        name: "Juno Updated",
+      },
+      relationships: {
+        articles: [
+          {
+            id,
+          },
+        ],
+      },
+    })
+    const updatedAdminUserQuery = await hatchedReactRest.Admin_User.findOne(
+      userId,
+    )
+    expect(updatedAdminUserQuery).toEqual({
+      id: userId,
+      __schema: "Admin_User",
+      name: "Juno Updated",
+    })
```

Comment:
this test should do findOne with include on articles to make sure that the articles id matches the ones on the `updateOne` call on line 269